cogs/ranked.py

```diff
@@ -651,6 +651,38 @@ async def leave(self, interaction: discord.Interaction, game: str):
             f"Queue for __{qdata.full_game_name}__ is now **[{qdata.queue.qsize()}/{qdata.game_size}]**",
             delete_after=60)
 
+    @app_commands.choices(game=games_choices)
+    @app_commands.command(name="leaveall", description="Remove yourself from all queues")
+    async def leaveall(self, interaction: discord.Interaction):
+        logger.info(f"{interaction.user.name} called /leaveall")
+
+        ephemeral = False
+
+        if (isinstance(interaction.channel, discord.TextChannel) and
+                isinstance(interaction.user, discord.Member) and
+                interaction.channel.id == QUEUE_CHANNEL):
+            player = interaction.user
+            dequeued = []
+            for qdata in game_queues[game]:
+                if player in qdata.queue:
+                    qdata.queue.remove(player)
+                    await self.update_ranked_display()
+                    cleaned_display_name = ''.join(char for char in player.display_name if char.isalnum())
+                    message = f"🔴 **{cleaned_display_name}** 🔴\nremoved from the queue for __{qdata.full_game_name}__. *({qdata.queue.qsize()}/{qdata.game_size})*"
+                    dequeued.append(qdata)
+            if (len(dequeued) == 0):
```

Comment:
This piece is good - perhaps add an else statement to this to address some of the notes made above

Reply:
Sorry, but I'm not sure what the else statement would need to adjust